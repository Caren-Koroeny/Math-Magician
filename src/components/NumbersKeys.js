// snippet rce
import React, { Component } from 'react';

class NumbersKeys extends Component {
  // snippet rconst
  constructor(props) {
    super(props);

    this.state = {

    };
  }

  render() {
    return (
      <div />
    );
  }
}

export default NumbersKeys;
