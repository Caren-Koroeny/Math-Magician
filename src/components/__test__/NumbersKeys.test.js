import React from 'react';
import renderer from 'react-test-renderer';
import NumbersKeys from '../NumbersKeys';

it('renders operations correctly', () => {
  const tree = renderer.create(<NumbersKeys />);
  expect(tree).toMatchSnapshot();
});
