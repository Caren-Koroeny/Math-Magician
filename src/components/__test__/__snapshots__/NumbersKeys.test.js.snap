// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders operations correctly 1`] = `
<div
  className="allNumbers"
>
  <button
    className="btn btn-primary"
    type="button"
  >
    7
  </button>
  <button
    className="btn btn-primary"
    type="button"
  >
    8
  </button>
  <button
    className="btn btn-primary"
    type="button"
  >
    9
  </button>
  <button
    className="btn btn-primary"
    type="button"
  >
    4
  </button>
  <button
    className="btn btn-primary"
    type="button"
  >
    5
  </button>
  <button
    className="btn btn-primary"
    type="button"
  >
    6
  </button>
  <button
    className="btn btn-primary"
    type="button"
  >
    1
  </button>
  <button
    className="btn btn-primary"
    type="button"
  >
    2
  </button>
  <button
    className="btn btn-primary"
    type="button"
  >
    3
  </button>
  <button
    className="btn btn-primary col-span-2"
    type="button"
  >
    0
  </button>
  <button
    className="btn btn-primary"
    type="button"
  >
    .
  </button>
</div>
`;
